src/overview/reducer.js

```diff
@@ -198,6 +202,19 @@ export default createReducer(
             ...state,
             activeTagIndex: defaultState.activeTagIndex,
         }),
+        [actions.resetSharePopup]: state => ({
+            ...state,
+            sharePopup: defaultState.sharePopup,
+        }),
+        [actions.falsecopy]: state => ({
```

Comment:
You can set the value of `copy` by using by taking value in the argument.

    [actions.setCopy]: (state, copy) => ({
        ...state,
        copy: copy
    })

Reply:
Fixed